test/sagemaker_tests/pytorch/training/resources/mnist/smdataparallel_mnist.py

```diff
@@ -20,6 +20,7 @@
 import torch.nn.functional as F
 import torch.optim as optim
 import torchvision
+from packaging import version
```

Comment:
nit: Please use `from packaging.version import Version`, as the `Version` class is more intuitive in comparing version strings than `version.parse`.